packages/editor/src/client/panel/authorEditPanel.tsx

```diff
@@ -220,7 +220,9 @@ export function AuthorEditPanel({id, onClose, onSave}: AuthorEditPanelProps) {
             </ListInput>
           </Panel>
           <Panel header={t('authors.panels.bioInformation')}>
-            <RichTextBlock value={bio} onChange={value => setBio(value)} />
+            <div className="richTextFrame">
```

Comment:
Since this looks like a reusable and practical feature, can you put it inside the RichTextBlock component as an optional prop?

Reply:
@penk85 we're rebuilding the whole richTextBlock with the dreifuss-wysiwyg-editor and will make the changes once we implement it.